service/src/main/java/uk/gov/hmcts/reform/pebble/TodayFilter.java

```diff
@@ -31,6 +30,6 @@ public TodayFilter(Clock clock) {
 
     @Override
     public Object apply(Object input, Map<String, Object> args) {
-        return Date.from(ZonedDateTime.now(this.clock).toInstant());
+        return java.util.Date.from(ZonedDateTime.now(this.clock).toInstant());
```

Comment:
Could probably still be a static import, just not from the sql library this time

Reply:
of course:) done